packages/google_sign_in/google_sign_in_web/pubspec.yaml

```diff
@@ -22,8 +22,11 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
+  google_identity_services_web: ^0.2.0
   google_sign_in_platform_interface: ^2.2.0
+  http: ^0.13.5
   js: ^0.6.3
+  jwt_decoder: ^2.0.1
```

Comment:
This decoder is different from the used on testing (`package:jose`) because it has no extra dependencies.

Reply:
Same question here; is there a reasonable way to not add this dependency?

Reply:
Turns out JWT parsing is not that hard, so I've reimplemented it in a couple of helper functions, rather than importing this library.